PluginAutotaskTest/Plugin/PluginIntegrationTest.cs

```diff
@@ -234,6 +245,68 @@ public async Task DiscoverSchemasRefreshTest()
             await server.ShutdownAsync();
         }
 
+        [Fact]
+        public async Task DiscoverSchemasUserDefinedTest()
+        {
+            // setup
+            Server server = new Server
+            {
+                Services = {Publisher.BindService(new PluginAutotask.Plugin.Plugin())},
+                Ports = {new ServerPort("localhost", 0, ServerCredentials.Insecure)}
+            };
+            server.Start();
+
+            var port = server.Ports.First().BoundPort;
+
+            var channel = new Channel($"localhost:{port}", ChannelCredentials.Insecure);
+            var client = new Publisher.PublisherClient(channel);
+
+            var connectRequest = GetConnectSettings();
+
+            var query = @"BillingCodes
+{""Filter"":[{""field"":""Id"",""op"":""gte"",""value"":0}]}";
```

Comment:
The method for querying looks straight forward, but do the users know what the format of the second line will be? There might be an opportunity to provide an example format or something like that.